### PR TITLE
cchs 1.3.2

### PR DESCRIPTION
**Pull request by wyusuf068**
Upcoming resubmission to CRAN. Package passes build checks

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 ^README.html
 ^CRAN-RELEASE$
 ^CODE_OF_CONDUCT.md
+^.github
```
Comment (DougManuel) [inline .Rbuildignore]: Thumbs up.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
```
Comment (DougManuel) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Let's take out the references for smart phones.

Comment (wyusuf068) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Just for clarification, remove the section for smartphone?

Comment (DougManuel) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes. references like iPhone 6, iOS. We can remove the whole smartphone section.

File: cran-comments.md [hunk 3]
```diff
@@ -1,29 +1,12 @@
 ## Resubmission
 
 This is a resubmission. In this version we have:
-* Changed `print_note()` parameter in `rec_with_table()` to `notes`.
-* Enabled the use of message() to suppress messages on console
-* Added quotation marks to 'sjmisc' in `DESCRIPTION`
-* Added food_insecurity_der as a new derived variable
+* Added removed space in doi specifications in `DESCRIPTION`
```
Comment (DarkShadeKnigh) [inline cran-comments.md]: no need for added just keep removed

File: cran-comments.md [hunk 4]
```diff
@@ -1,29 +1,12 @@
...
+* Added binge_drinker as a new derived variable
+* Done minor edits to the labels and sections in `variables.csv`
+* Done minor edits to the documentation of `resp_condition_fun1()`,
```
Comment (DarkShadeKnigh) [inline cran-comments.md]: Replace "Done" with "Made"?

**Pull request merged**